packages/blocks/src/lib/navigation-map/components/robot-marker.svelte

```diff
@@ -1,14 +1,23 @@
 <script lang="ts">
-import { theme } from '@viamrobotics/prime-core/theme';
 import { MapLibreMarker, type LngLat } from '$lib';
 
+export let robotIcon = document.createElement('div');
```

Comment:
Why make this a div with a background image and not just an SVG?

Reply:
I found this [marker.ts](https://github.com/maplibre/maplibre-gl-js/blob/2fa582c90/src/ui/marker.ts#L152) file.
From it I learned that the icon of a marker is from its element. If when a marker is created and its element is undefined the default element is same as the one used by waypoints.

My reasoning is that using a div with background image allows us to supplant the default element.